src/Symfony/Component/HttpKernel/HttpCache/Esi.php

```diff
@@ -84,7 +84,7 @@ public function hasSurrogateCapability(Request $request)
     public function addSurrogateCapability(Request $request)
     {
         $current = $request->headers->get('Surrogate-Capability');
-        $new = 'symfony2="ESI/1.0"';
+        $new = 'symfony3="ESI/1.0"';
```

Comment:
Nobody should rely on the name, but @a-ast can you add an entry in the CHANGELOG/UPGRADE files to document that change?